vite.config.js

```diff
@@ -1,8 +1,41 @@
+/* eslint-disable no-undef */
```

Comment:
Let's try not to disable eslint guard rails ;) 

A solution could be to do something like this instead:

```suggestion
import * as path from 'path';
import { fileURLToPath } from 'url';

const __dirname = path.dirname(fileURLToPath(import.meta.url));
```

this way you don't have to turn of undefined checking for the whole file.